### PR TITLE
Prepared Storage for release

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.6.0-beta.4 (Unreleased)
-
+## 12.6.0-beta.4 (2021-05-12)
+- This release contains bug fixes to improve quality.
```
Comment (gapra-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md:4]: Should we mention adds support for service version 2020-08-04? Same comment for rest of the packages

**Review by kasobol-msft — APPROVED** [review]

**Comment by gamnguyen1013** [comment]
Hello, I'm using Azure.Storage.Blobs.Batch version 12.5.2 and try to SetBlobsAccessTierAsync. The error with status code 400, errorCode: InvalidAuthenticationInfo. Message: (Authentication information is not given in the correct format. Check the value of Authorization header.). I checked BatchRestClient and didn't see we add Authorization header. And I see it is a required header https://docs.microsoft.com/en-us/rest/api/storageservices/blob-batch#authorization. Can you please give me a suggestion what I should do in this case?

**Comment by amnguye** [comment]
Hi, This is not the correct place to submit an issue or query. Please go to the issues tab and submit your issue there with how to reproduce your issue, as this is not enough information to go off of. Thanks.

**Pull request merged**